lib/wp2middleman/frontmatter.rb

```diff
@@ -10,7 +10,7 @@ def initialize(post, include_fields: [])
     def post_data
       data = {
         'title' => post.title,
-        'date' => post.date_published,
+        'date' => post.date_time_published,
```

Comment:
If `#post_data` now uses `#date_time_published`, rather than `#date_published`, should `#valid?` also be changed to evaluate `#date_time_published` rather than `#date_published`?